### ISSUE TITLE
Datasets for other PLs

### ISSUE THREAD

**Issue by AlpacaMax** [oscarkarnalim/sourcecodeplagiarismdataset#2]
Hi. The dataset you guys made is great. However, since Python3, C, and C++ are also widely used in CS assignments, do you guys have any plans to make datasets for these languages as well?

**Comment by oscarkarnalim** [comment]
Yes, we plan to do similar thing for Python next year. The project is slightly delayed due to pandemic. I will keep you posted. In case it is urgent, you can translate the Java data set to other programming languages. You can push the translated data set here also if you want. I am happy to merge it with mine.
